### ISSUE TITLE
[Bug]: load timeout(180s) for a small data size collection with one pod standalone mode

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#36532]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20240925-aee046e5-amd64
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

It is a small data size collection, it should not be timeout

```

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:11 - INFO - ci_test]: assert insert: 0.22279667854309082 (test_e2e.py:45)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:11 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:21 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:21 - INFO - ci_test]: [test][2024-09-25T06:38:11Z] [10.16420218s] e2e__uGRvR4KU flush -> None (wrapper.py:30)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:21 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:31 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:31 - INFO - ci_test]: [test][2024-09-25T06:38:21Z] [10.16485694s] e2e__uGRvR4KU flush -> None (wrapper.py:30)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:31 - DEBUG - ci_test]: (api_request)  : [Collection.flush] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:41 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:41 - INFO - ci_test]: [test][2024-09-25T06:38:31Z] [10.16469345s] e2e__uGRvR4KU flush -> None (wrapper.py:30)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:41 - INFO - ci_test]: assert flush: 20.331419944763184, entities: 2000 (test_e2e.py:55)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:38:41 - DEBUG - ci_test]: (api_request)  : [Collection.load] args: [None, 1, 180], kwargs: {} (api_request.py:62)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:41:41 - ERROR - pymilvus.decorators]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=wait for loading collection timeout, collection: e2e__uGRvR4KU)>, <Time:{'RPC start': '2024-09-25 06:38:41.796785', 'RPC error': '2024-09-25 06:41:41.830532'}> (decorators.py:140)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:41:41 - WARNING - pymilvus.decorators]: Retry timeout: 180s (decorators.py:106)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:41:41 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=1, message=Retry timeout: 180s, message=wait for loading collection timeout, collection: e2e__uGRvR4KU)>, <Time:{'RPC start': '2024-09-25 06:38:41.792106', 'RPC error': '2024-09-25 06:41:41.830906'}> (decorators.py:140)

[2024-09-25T06:41:50.166Z] [2024-09-25 06:41:41 - ERROR - ci_test]: Traceback (most recent call last):

[2024-09-25T06:41:50.166Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 32, in inner_wrapper

[2024-09-25T06:41:50.167Z]     res = func(*args, **_kwargs)

[2024-09-25T06:41:50.167Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 63, in api_request

[2024-09-25T06:41:50.167Z]     return func(*arg, **kwargs)

[2024-09-25T06:41:50.167Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 429, in load

[2024-09-25T06:41:50.167Z]     conn.load_collection(

[2024-09-25T06:41:50.167Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 141, in handler

[2024-09-25T06:41:50.167Z]     raise e from e

[2024-09-25T06:41:50.167Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 137, in handler

[2024-09-25T06:41:50.167Z]     return func(*args, **kwargs)

[2024-09-25T06:41:50.167Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 176, in handler

[2024-09-25T06:41:50.167Z]     return func(self, *args, **kwargs)

[2024-09-25T06:41:50.167Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 107, in handler

[2024-09-25T06:41:50.167Z]     raise MilvusException(

[2024-09-25T06:41:50.167Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=Retry timeout: 180s, message=wait for loading collection timeout, collection: e2e__uGRvR4KU)>

[2024-09-25T06:41:50.167Z]  (api_request.py:45)

[2024-09-25T06:41:50.167Z] [2024-09-25 06:41:41 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=Retry timeout: 180s, message=wait for loading collection timeout, collection: e2e__uGRvR4KU)> (api_request.py:46)
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/chaos-test-cron/detail/chaos-test-cron/18925/pipeline
log:
[artifacts-one-pod-standalone-pod-kill-18925-server-logs.tar.gz](https://github.com/user-attachments/files/17141969/artifacts-one-pod-standalone-pod-kill-18925-server-logs.tar.gz)

**Comment by zhuwenxing** [comment]
another failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/chaos-test-cron/detail/chaos-test-cron/18940/pipeline
log:
[artifacts-one-pod-standalone-pod-failure-18940-server-logs.tar.gz](https://github.com/user-attachments/files/17142010/artifacts-one-pod-standalone-pod-failure-18940-server-logs.tar.gz)


so this issue seems like a stable issue.

**Comment by zhuwenxing** [comment]
still reproduced in master-20241101-f190e5d8-amd64
failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/chaos-test-cron/detail/chaos-test-cron/19322/pipeline

log:
[artifacts-one-pod-standalone-pod-failure-19322-server-logs.tar.gz](https://github.com/user-attachments/files/17599176/artifacts-one-pod-standalone-pod-failure-19322-server-logs.tar.gz)

**Comment by tedxu** [comment]
Something is wrong with loading of index file, according to the log:

```
2024-11-01 16:46:36.894 | I20241101 08:46:36.894152  1699 load_index_c.cpp:266] [SERVER][AppendIndexV2][milvus] [collection=453627208604844603][segment=453627208605244681][field=102][enable_mmap=false] load index 453627208604844613 |  
-- | -- | --
  |   | 2024-11-01 16:46:36.894 | [2024-11-01T08:46:36Z INFO  tantivy::indexer::segment_updater] save metas |  
  |   | 2024-11-01 16:46:36.896 | E20241101 08:46:36.896276  1699 DiskFileManagerImpl.cpp:254] [SERVER][CacheIndexToDisk][milvus] invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi |  
  |   | 2024-11-01 16:46:36.896 | [2024/11/01 08:46:36.896 +00:00] [WARN] [segments/load_index_info.go:200] ["CStatus returns err"] [traceID=03159d61a8f8137905f2fa4b19851efa] [error="invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi"] [extra="AppendIndex failed"] |  
  |   | 2024-11-01 16:46:36.896 | [2024/11/01 08:46:36.896 +00:00] [WARN] [segments/segment.go:1218] ["load index failed"] [error="invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi"] |  
  |   | 2024-11-01 16:46:36.904 | [2024/11/01 08:46:36.904 +00:00] [WARN] [segments/segment_loader.go:322] ["load segment failed when load data into memory"] [traceID=03159d61a8f8137905f2fa4b19851efa] [collectionID=453627208604844603] [segmentType=Sealed] [requestSegments="[453627208605244681]"] [preparedSegments="[453627208605244681]"] [partitionID=453627208604844604] [segmentID=453627208605244681] [segmentType=L1] [error="At LoadSegment: invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi"] [errorVerbose="At LoadSegment: invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi\n(1) attached stack trace\n  -- stack trace:\n  \| github.com/milvus-io/milvus/internal/querynodev2/segments.(*segmentLoader).Load.func5\n  \| \t/workspace/source/internal/querynodev2/segments/segment_loader.go:335\n  \| github.com/milvus-io/milvus/pkg/util/funcutil.ProcessFuncParallel.func3\n  \| \t/workspace/source/pkg/util/funcutil/parallel.go:86\n  \| runtime.goexit\n  \| \t/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.22.0.linux-amd64/src/runtime/asm_amd64.s:1695\nWraps: (2) At LoadSegment\nWraps: (3) invalided index file path:/var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset, error:stoi\nError types: (1) *withstack.withStack (2) *errutil.withPrefix (3) merr.milvusError"]

```

**Comment by tedxu** [comment]
@smellthemoon please take a look.

**Comment by xiaofan-luan** [comment]
can we reproduce with https://github.com/milvus-io/milvus/commit/2ff3765058d175178aff4209860df37232828349 this pr in So we can get more details

**Comment by xiaofan-luan** [comment]
we need to get /var/lib/milvus/data/index_files/453627208605245387/1/453627208604844604/453627208605244681/index_null_offset
file and parser

**Comment by smellthemoon** [comment]
The rootpath of chunk manager on the c side does not process the trailing /, resulting in inconsistent file paths on both sides, I will fix it

**Comment by smellthemoon** [comment]
pr merged, could you please help to verify? @zhuwenxing

**Comment by zhuwenxing** [comment]
verified and fixed in `master-20241111-a88fff13-amd64`
